src/mongocrypt.h.in

```diff
@@ -662,6 +662,13 @@ mongocrypt_ctx_setopt_masterkey_local (mongocrypt_ctx_t *ctx);
  *    provider: "local"
  * }
  * 
+ * KMIP
+ * {
+ *    provider: "kmip",
+ *    endpoint: <string>
+ *    keyId: <optional string>
```

Comment:
```suggestion
 *    keyId: <optional string>
 *    endpoint: <string>
```
May not be necessary, but suggesting reordering of fields here to match order in `_mongocrypt_kmip_kek_t`.